build.gradle

```diff
@@ -56,7 +56,7 @@ repositories {
 }
 
 ext {
-    opensearch_version = System.getProperty("opensearch.version", "1.3.0-SNAPSHOT")
+    opensearch_version = System.getProperty("opensearch.version", "1.4.0-SNAPSHOT")
```

Comment:
should we replace this with the next version?

Reply:
The versions are a bit diverged. On main branch, security plugin is 2.0.0.0, but OpenSearch core version is 1.3.0